recipes/mamba-ssm/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "mamba-ssm" %}
+{% set version = "2.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mamba_ssm-{{ version }}.tar.gz
+  sha256: 5055c8e631a22bb3cfc9958828a7a57ea14deef2ab2a98dcf425a27c87dcf6a7
+
+build:
+  number: 0
+  skip: true   # [cuda_compiler_version == "None"]
+  script:
+    - export MAMBA_FORCE_BUILD=TRUE
+    - export MAMBA_FORCE_CXX11_ABI=TRUE
+    - export MAMBA_SKIP_CUDA_BUILD=FALSE
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cuda') }}
+  host:
+    - python
+    - setuptools >=61.0.0
+    - wheel
+    - pytorch
+    - packaging
+    - ninja
+    - pip
+    - cuda-cudart-dev
+  run:
+    - python
+    - pytorch
+    - triton
+    - ninja
```

Comment:
looks like this is also listed as a runtime dep upstream but really shouldnt be, i will try to fix upstream